### PR TITLE
Add "Bottle Song" exercise and test suite

### PR DESCRIPTION
**Pull request by rabestro**
A new exercise called "Bottle Song" has been introduced. This includes corresponding test suite for the exercise. Modification was also made to update the average run time value in the config file. Overall, this commit provides a new practice exercise for users to solve.

### CODE REVIEW

**Comment by rabestro** [comment]
I have an error message, so I fixed the number to 2.0

```shell
PS /Users/jegors/Projects/Exercism/awk> ./bin/configlet lint
The lint command is under development.
To check your track using the latest linting rules,
please regularly update configlet and re-run this command.

The value of `test_runner.average_run_time` is `2`, but it must be a float:
/Users/jegors/Projects/Exercism/awk/config.json

Configlet detected at least one problem.
For more information on resolving the problems, please see the documentation:
https://exercism.org/docs/building/configlet/lint
```

File: config.json [hunk 1]
```diff
@@ -585,6 +585,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 8
+      },
```
Comment (IsaacG) [inline config.json:588]: Please sort entries by difficulty then name.

**Comment by glennj** [comment]
> I have an error message, so I fixed the number to 2.0

upgrade configlet with `bin/fetch-configlet`

**Comment by glennj** [comment]
> I have an error message, so I fixed the number to 2.0

And now CI is unhappy ("The value of `test_runner.average_run_time` is `2.0`, but it must be an integer")

Upgrade your configlet: `bin/fetch-configlet`

**Comment by glennj** [comment]
> There should be exactly one commented skip directive in test-bottle-song.bats

The first test should contain
```bash
    #[[ $BATS_RUN_SKIPPED == "true" ]] || skip
```
and every other test should contain
```bash
    [[ $BATS_RUN_SKIPPED == "true" ]] || skip
```

**Comment by rabestro** [comment]
> ```shell
> [[ $BATS_RUN_SKIPPED == "true" ]] || skip
> ```
Yeah... I have to improve my prompt :)

image

**Comment by rabestro** [comment]
> There should be exactly one commented skip directive in test-bottle-song.bats

AI generated the whole test but I provide wrong sample with deleted comments. :)

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
> Based on the provided test data and the sample solution,

That's interesting. I start by translating the canonical data to create the tests (I use vim, and `q` recording macros takes care of a most of the repetitive grunt work), and then I write the example solution.

**Pull request merged**